### ISSUE TITLE
Problem on TV shows

### ISSUE THREAD

**Issue by tiagomiguel1994** [123Venom/plugin.video.venom#132]
Good afternoon
I'm having a problem with TV shows, I don't know if it's a problem with Kodi or the theme, but this has happened recently... 
On certain TV shows like Squid Game or Game of Thrones when selecting the episode I always get a message, "no stream available" but for example in The Witcher or Walking Dead series the streams already appear.  
This happens on NVIDIA shield Pro, because for example in the internal TV applications I also have the Venom there and everything works perfectly... 
Another bug, but this one is in the 2 devices and it is in a particular series. 
In the Money Heist series there is there 3 seasons but the episodes don't match up with the titles and with the background image that has each episode. 
I'm writing here but I'm not sure if the problem is with Venom.  
Greetings

**Comment by drinfernoo** [comment]
Regarding Money Heist or Squid Game, I imagine those may be tricky since the originals were in a different language.

Specifically, Money Heist might have an issue as it's been primarily distributed by Netflix, which has a tendency to mark their seasons oddly (Season One: Part Two, and the like), which has historically caused mismatches between scene releases and metadata providers.

**Comment by tiagomiguel1994** [comment]
> Regarding Money Heist or Squid Game, I imagine those may be tricky since the originals were in a different language.
> 
> Specifically, Money Heist might have an issue as it's been primarily distributed by Netflix, which has a tendency to mark their seasons oddly (Season One: Part Two, and the like), which has historically caused mismatches between scene releases and metadata providers.

Yes, I understand.  but the problem of "no streams available" I can't figure out where the problem is. And is has nothing to do with being original or not.

**Comment by drinfernoo** [comment]
If the scraper can't find any matching streams, due to any reason (not just the ones I mentioned), then there won't be any "available"... The reasons I mentioned are just two of many possible causes for no matching streams being found.

**Comment by 123Venom** [comment]
@tiagomiguel1994 lets start off with a more common show you mentioned, **Game of Thrones**.  Give me a season and episode that returns a `no streams available` for you so I can test it.  I've watched a good deal of this show and never had an issue with that.  What filters are you using?  File size limits?  Default FenomScrapers in use? Also

>In the Money Heist series there is there 3 seasons but the episodes don't match up with the titles and with the background image that has each episode.
Screenshot this condition so I can see what your looking at.  I honestly felt a bit clueless from the lack of details given.  What background image?  You mean the fanart artwork?  Show level and season level all were the same to me so I have no idea what you are talking about...picture is worth a thousand words.  Scrape/Source Select results do not use background/fanart artwork so I assumed you can't be talking about that, but ran one check in just case.

<img width="1334" alt="Money_Heist_season3" src="https://user-images.githubusercontent.com/46383815/135935705-dc671ec7-cbc5-4889-a253-c13f15035e61.PNG">

<img width="1332" alt="Money_Heist_s03e01" src="https://user-images.githubusercontent.com/46383815/135935733-0cfcd8c2-8dd1-4e52-8902-77a5dcb1ee19.PNG">

**Comment by tiagomiguel1994** [comment]
It works now, for some reason fenom was out of date and had the previous version.
I went through the settings and checked in to notify me about new updates and today a message appeared that had a new version of fenom.
I went to the details of fenom and I forced the update, now everything has streams... but what I find strange is not updating fenom automatically, as automatic updates were turned on.
Now about Monei Heist, my theme shows a picture of the supposed episode, the name of the episode and the picture that you are going to look for is right, what's wrong is the stream.
This episode you have there that is the same as me I have. 
It belongs to season 2, if you go to see the S03 EP5 it is the same as the S02 EP5.  
But that's nothing you can do because it has to do with naming the titles... In my opinion on Venom this series had to be remodeled as it is on IMDB with 5 seasons, i think the problem was solved.  
BTW it's nothing that bothers me because I went to rarbg I put the magnet link on alldebrid and then via Venom I accessed the cloud storage... 
I want to thank you for the speed of the response, it continues like this... I see more and more people using this addon and love to use it 
Greetings

**Comment by 123Venom** [comment]
I've sorted out and fixed the Trakt list issue.  That will be fixed in the next version.

I'm not 100% sure I follow you but I did peak at TMDb vs. IMDb meta to compare for **Money Heist**.   I found an episode on TMDb S02E05 called **Boom, Boom, Ciao** that IMDb has as S03E05 and I believe this to be the issue you are talking about.  Venom indexes it's meta from TMDb so it is as THEY provide it.  I'm not mixing things up for a straggler show here or there that a meta provider has messed up...that's one is on them.  I had no idea what you mean by `stream`, there's no stream/source links in your 2nd pic.  What you can do is report discrepancies to TMDb on their forum.  I've done it numerous times.

**Comment by tiagomiguel1994** [comment]
I didn't know the data was obtained via TMDb.  
Yes I went to TMDb's website and it's really only been there 3 seasons.  
Regarding what you said, yes it is exactly that, on TMDb the S02E05 is on the IMDb the S03E05 which makes the plugin show the wrong episodes and not even show this entire last season... But forget it, it was like I said in the first thread I don't know if this is Venom's problem, I only took the opportunity to talk about this subject out of curiosity, because the main problem is solved.
Thank you very much for your availability.  
greetings
